react-components/stories/SceneContainer.stories.tsx

```diff
@@ -74,21 +115,322 @@ export const Main: Story = {
 type SceneContainerStoryContentProps = {
   sceneExternalId: string;
   sceneSpaceId: string;
+  defaultResourceStyling?: DefaultResourceStyling;
 };
 
 const SceneContainerStoryContent = ({
   sceneExternalId,
-  sceneSpaceId
+  sceneSpaceId,
+  defaultResourceStyling
 }: SceneContainerStoryContentProps): ReactElement => {
   const reveal = useReveal();
   const { fitCameraToSceneDefault } = useSceneDefaultCamera(sceneExternalId, sceneSpaceId);
```

Comment:
I know it's "just" a storybook - but this function is getting ridiculously long now :sweat_smile: Can we factor it out into a few smaller functions/hooks?